.github/workflows/stale-issue.yml

```
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2025 Canonical Ltd.
+name: Stale issue/pr
+
+on:
+  workflow_dispatch:
+    inputs:
+      days_before_stale:
+        description: 'Days before stale'
+        default: 120
+        required: true
+      days_before_close:
+        description: 'Days before close'
+        default: 15
+        required: true
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9.0.2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue has been stale for 120 days and will be closed in 15 days. Comment to keep it open.'
+          stale-pr-message: 'This pull request has been stale for 120 days and will be closed in 15 days. Comment to keep it open.'
```

Should not the text for the number of days ("120" and "15") be changed based on the workflow input?
```suggestion
          stale-issue-message: 'This issue has been stale for ${{ inputs.days_before_stale }} days and will be closed in ${{ inputs.days_before_close}} days. Comment to keep it open.'
          stale-pr-message: 'This pull request has been stale for ${{ inputs.days_before_stale }} days and will be closed in ${{ inputs.days_before_close}} days. Comment to keep it open.'
```

---

Good catch. Done.